src/mbgl/gl/shader.hpp

```diff
@@ -31,7 +32,7 @@ class Shader : private util::noncopyable {
            const char* vertex,
            const char* fragment,
            Context&,
-           Defines defines = Defines::None);
```

Comment:
Since `Defines` isn't used anymore, you should also remove the `enum` definition.